src/IO/H5/Python/File.cpp

```diff
@@ -16,6 +16,7 @@
 #include "IO/H5/OpenGroup.hpp"
 #include "IO/H5/Type.hpp"
 #include "IO/H5/Version.hpp"
+#include "IO/H5/Helpers.hpp"
```

Comment:
clang-tidy is complaining about that `H` is before `O`, `T` and `V` in the alphabet, and it has a valid point. If you run clang-format this should get corrected as well :)

Reply:
This still needs to be fixed